### PR TITLE
Setup Danger

### PR DESCRIPTION
**Pull request by MP0w**
let's see if this works!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ router.get("/articles/:id") { request in
 }
 ```
 
-But, of course, you could perform any logic which fits your needs:
+Of course you can perform any logic which fits your needs:
```
Comment (joanromano) [inline README.md:217]: ? For the lulz?

File: Dangerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+fail("fcontext left in tests") if `grep -r fcontext Tests/ `.length > 1
+fail("fit left in tests") if `grep -r fit Tests/ `.length > 1
+
+# If files are changed and test were not changed just warn
+has_app_changes = !git.modified_files.grep(/Source/).empty?
+has_test_changes = !git.modified_files.grep(/Tests/).empty?
+
+if has_app_changes && !has_test_changes
+  warn("Tests were not updated", sticky: false)
+end
```
Comment (joanromano) [inline Dangerfile:20]: Nice! What's the sticky for?

File: Dangerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+markdown_files = (git.added_files + git.modified_files).select{ |file| file.end_with? "md" }
+prose.lint_files markdown_files
```
Comment (joanromano) [inline Dangerfile]: What will the linter in markdown files do?

**Comment by joanromano** [comment]
👍

File: Dangerfile (hunk 3 above)
Comment (MP0w) [inline Dangerfile]: still not sure I will check when ci finish and see if I will keep it or kick it for the moment... seems to be checking the correctness and maybe spelling (for example it found a sentence started with But)

File: Dangerfile (hunk 2 above)
Comment (MP0w) [inline Dangerfile:20]: will be deleted once you fix it!

File: README.md (hunk 1 above)
Comment (MP0w) [inline README.md:217]: found by the linter :D

File: Dangerfile (hunk 3 above)
Comment (joanromano) [inline Dangerfile]: Ah now I understand why you changed it lol

**Pull request merged**